File: src/IsbInterop/ComponentTokens/Proxies/ComponentTokenInfo.vb
Imports IsbInterop.Base.Proxies

Namespace ComponentTokens.Proxies

  ''' <summary>
  ''' Обертка над IComponentTokenInfo.
  ''' </summary>
  Friend Class ComponentTokenInfo
    Inherits EdmsObjectInfo
    Implements IComponentTokenInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIComponentTokenInfo">COM-объект IComponentTokenInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwIComponentTokenInfo As Object, scope As IScope)
      MyBase.New(rcwIComponentTokenInfo, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop/ComponentTokens/Proxies/ComponentTokenInfo.cs
using IsbInterop.Base.Proxies;

namespace IsbInterop.ComponentTokens.Proxies
{
  /// <summary>
  /// Обертка над IComponentTokenInfo.
  /// </summary>
  internal class ComponentTokenInfo : EdmsObjectInfo, IComponentTokenInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIComponentTokenInfo">COM-объект IComponentTokenInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public ComponentTokenInfo(object rcwIComponentTokenInfo, IScope scope)
      : base(rcwIComponentTokenInfo, scope) { }
  }
}
